nearlib/test/integration.test.js

```diff
@@ -21,6 +25,69 @@ test('test creating default config', async () => {
     Near.createDefaultConfig();
 });
 
+describe("dev connect", () => {
+    test('test dev connect with no account creates a new account', async () => {
+        const keyStoreNoAccount = new InMemoryKeyStore();
+        const localStorage = createMockLocalStorage();
+        const nearNoAccount = await dev.connect({
+            nodeUrl: 'http://localhost:3030',
+            deps: {
+                keyStore: keyStoreNoAccount,
+                localStorage: localStorage,
+                createAccount: (async (newAccountName, newAccountPublicKey) => {
+                    const createAccountResponse = await account.createAccount(newAccountName, newAccountPublicKey, 1, aliceAccountName);
+                    await nearjs.waitForTransactionResult(createAccountResponse);
+                })
+            },
+        });
+        expect(Object.keys(keyStoreNoAccount.keys).length).toEqual(1);
+        const newAccountId = Object.keys(keyStoreNoAccount.keys)[0];
+        const viewAccountResponse = await account.viewAccount(newAccountId);
+        const newAccountKeyPair = await keyStoreNoAccount.getKey(newAccountId);
+        expect(newAccountKeyPair).toBeTruthy();
+        const expectedAccount = {
+            nonce: 0,
+            account_id: newAccountId,
+            amount: 1,
+            code_hash: newAccountCodeHash,
+            stake: 0,
+        };
+        expect(viewAccountResponse).toEqual(expectedAccount);
+        expect(localStorage.getItem(localStorageAccountIdKey)).toEqual(newAccountId);
+    });
+
+    
+    test('test dev connect with invalid account creates a new account', async () => {
```

Comment:
these 2 tests are quite long and very hard to compare (as most code is the same, but not really).

is there any way to move common initialization to `before` / `beforeAll`?

Reply:
some, done